code/_helpers/unsorted.dm

```diff
@@ -821,6 +821,9 @@ proc/GaussRandRound(var/sigma,var/roundto)
 					for(var/mob/M in T)
 						if(istype(M, /mob/observer/eye)) continue // If we need to check for more mobs, I'll add a variable
 						M.loc = X

```

Comment:
Would it be better to replace this line with ``M.forceMove(X)``, since the comment you made below sounds like it would also fix it, or is there a reason forceMove cant be used?

Reply:
The comment above it says it shouldn't use forceMove- I immediately went to change it, but decided against it because of the comment.

Reply:
Heeey you found my comment.

It's debatable on whether or not you should use forceMove(). This calls things like entered, crossed, etc on all the things that are moving. Technically things in a whole area that just move DIDN'T move relative to their surroundings, thus I just set loc instead.

Like if you, for example, have ambient elevator music like we do, using forceMove resets the music every time the turbolift moves because you "moved". But in actuality, you're not supposed to have "moved", the whole lift/shuttle/whatever moved with you, so setting loc seems to represent this better.

Reply:
I can buy that explanation.